<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="line_id">
       <property name="placeholderText">
        <string>ID (При создании оставить пустым)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_name">
       <property name="placeholderText">
        <string>Название</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_roast">
       <property name="placeholderText">
        <string>Степень обжарки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_type">
       <property name="placeholderText">
        <string>Тип(молотый/в зернах)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_taste">
       <property name="placeholderText">
        <string>Описание вкуса</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_cost">
       <property name="placeholderText">
        <string>Цена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_size">
       <property name="placeholderText">
        <string>Объем упаковки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_add">
       <property name="text">
        <string>Добавить/редактировать</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
